docs/test_config.md

```diff
@@ -0,0 +1,24 @@
+# Test config
+
+Sometimes, we want to scale up or down our tests to compare against the size of a given architecture.
```

Comment:
While you have added some text incorporating the suggested change this line should be updated too.   It is misleading as the opener of the test config section as it's not just variables that track with the size of an architecture that the test configuration allows a user to control (ex: expected successes, number of iterations)